apps/api/src/domain/models/context/test-data/builders/build-resources-connected-with-note.ts

```diff
@@ -0,0 +1,59 @@
+import { AggregateType, LanguageCode } from '@coscrad/api-interfaces';
+import { clonePlainObjectWithOverrides } from '../../../../../lib/utilities/clonePlainObjectWithOverrides';
+import { EventMetadataBuilder } from '../../../../../test-data/events';
+import { DeepPartial } from '../../../../../types/DeepPartial';
+import buildDummyUuid from '../../../__tests__/utilities/buildDummyUuid';
+import {
+    ResourcesConnectedWithNote,
+    ResourcesConnectedWithNotePayload,
+} from '../../commands/connect-resources-with-note/resources-connected-with-note.event';
+import { GeneralContext } from '../../general-context/general-context.entity';
+import { TimeRange, TimeRangeContext } from '../../time-range-context/time-range-context.entity';
+import { EdgeConnectionContextType } from '../../types/EdgeConnectionContextType';
+
+export const buildResourcesConnectedWithNote = (
+    payloadOverrides: DeepPartial<ResourcesConnectedWithNotePayload>,
+    buildMetadata: EventMetadataBuilder
+) => {
+    const defaultPayload: ResourcesConnectedWithNotePayload = {
+        aggregateCompositeIdentifier: {
+            type: AggregateType.note,
+            id: buildDummyUuid(1),
+        },
+        text: 'this part of the video relates to the book',
+        languageCode: LanguageCode.Haida,
+        fromMemberCompositeIdentifier: {
+            type: AggregateType.video,
+            id: buildDummyUuid(3),
+        },
+        fromMemberContext: new TimeRangeContext({
+            type: EdgeConnectionContextType.timeRange,
+            timeRange: new TimeRange({
+                inPointMilliseconds: 100,
```

Comment:
Extremely short time range!

Reply:
All that matters is that it's bigger than 0 for the purpose of this test. Only the "COSCRAD type" matters for this one.